modules/scripting/src/main/java/org/mule/module/scripting/builders/ScriptConfigurationBuilder.java

```diff
@@ -66,6 +67,9 @@ public ScriptConfigurationBuilder(String scriptEngineName, String[] configResour
     protected void doConfigure(MuleContext muleContext) throws Exception
     {
         this.muleContext = muleContext;
+        SimpleRegistry registry = new SimpleRegistry(muleContext);
```

Comment:
This related to my previous comments.  IMO, when spring-config isn't on the classpath muleContext should still be ready to use without having to manually create, initialise and add a SimpleRegisty.